ca_on_toronto/helpers.py

```diff
@@ -11,16 +11,23 @@ def committees_from_sessions(self, sessions=[]):
         for a in page.xpath('//table[@id="list"]//td[@class="db"]/a'):
             link = a.attrib['href']
             data = committee_from_url(self, link)
-            data.update({'term': term})
+            if data is not None:
+                data.update({'term': term})
 
-            yield data
+                yield data
 
 
 def committee_from_url(self, url=None):
     page = self.lxmlize(url)
     script_text = page.xpath('//head/script[not(@src)]/text()')[0]
     committee_name = re.search(r'var decisionBodyName = "(.*)";', script_text).group(1)
-    committee_code = re.search(r'meetingRefs\.push\("[0-9]{4}\.([A-Z]{2})[0-9]+"\);', script_text).group(1)
+    cc_check = re.search(r'meetingRefs\.push\("[0-9]{4}\.([A-Z]{2})[0-9]+"\);', script_text)
+    if cc_check is not None:
```

Comment:
Can you add a comment (and include a sample URL) explaining why the check would sometimes fail?